go.mod

```diff
@@ -12,10 +12,10 @@ require (
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20211102075456-ffc4e11dfb16
 	github.com/IBM-Cloud/power-go-client v1.1.5
 	github.com/IBM/go-sdk-core/v4 v4.9.0
-	github.com/IBM/go-sdk-core/v5 v5.9.1
+	github.com/IBM/go-sdk-core/v5 v5.9.5
 	github.com/IBM/networking-go-sdk v0.14.0
 	github.com/IBM/platform-services-go-sdk v0.18.16
-	github.com/IBM/vpc-go-sdk v1.0.1
+	github.com/IBM/vpc-go-sdk v0.20.0
```

Comment:
Is the downgrade needed? Is quite a version jump.
Also vendor changes should be done in their own commit.

Reply:
@r4f4 this is not downgrade they have specified this as new version

Reply:
Indeed [1], that's confusing.

[1] https://github.com/IBM/vpc-go-sdk#ibm-cloud-vpc-go-sdk-version-0200